Source/WebCore/PAL/pal/spi/cocoa/QuartzCoreSPI.h

```diff
@@ -57,6 +57,10 @@
 #import <QuartzCore/CAFenceHandle.h>
 #endif
 
+#if PLATFORM(VISION)
+#import <QuartzCore/CARemoteEffectPrivate.h>
```

Comment:
Is there anything we're using from this private header that can/should be declared in the open source section below? The things you added below don't really belong here, since they are not in this header or QuartzCore.